article/_posts/2019-10-16-PickAndLearn.md

```diff
@@ -0,0 +1,90 @@
+classification tasks.
+
+
+# Methods
+The overall proposed end-to-end architecure is detailed in Figure 2.  The authors used the dice loss as segmentation
+loss and U-Net as their segmentation network.
+
+![](/article/images/PickAndLearn/figure2.png)
+
+
+## Quality Awareness Module (QAM)
+The QAM runs parallel with the segmentation module to calculate weight values to apply to the loss from each sample.
+During training, the re-weighted loss is used for both the segmentation module and the QAM.
+
+The network used for the QAM is a VGG network, where the number of input channel is change to $$n + 1$$, where $$n$$ is
+the number of segmentation classes, to take both image and labels as input. The final layer is also replaced by a
+one-channel average pooling layer to assess the weight for each sample in the mini-batch.
+
+The resulting loss used by the segmentation model then becomes:
+
+![](/article/images/PickAndLearn/equation1.png)
+
+The authors mention that they apply a softmax (which does not seem to be pictured in the equation) to rescale all the
+weights between 0 and 1.
+
+
+## Overfitting Control Module (OCM)
+The motivation behind the ÒCM is that overfitting problems can arise from the use of the QAM, if the QAM has a larger
+weight on a small subset with clear-labeled samples. The authors also state that if the QAM makes mistakes during the
+training process, the relative weight of this sample can be too small or too big for the network to correct.
+
+To counter this, the authors add an extra restriction between the quality awareness module and the softmax layer. They
+simply use a weighted hyperbolic tangent (as shown in the equation below), to limit the relative ratio of the output
+quality score from (−∞,∞) to (−λ,λ). The authors state they empirically set λ to be 2.
+
+![](/article/images/PickAndLearn/equation2.png)
+
+
+## Data
+The authors tested their method on the public JSRT dataset, which contains 247 X-rays images of the chest. The labeled
+structures are the heart, clavicles, and lungs.
+
+To generate noisy-labeled images, the authors used different types and levels of noises. They randomly selected
+0%, 25%, 50% and 75% samples from the training set and eroded or dilated them with 1 ≤ n i ≤ 8 and 5 ≤ n i ≤ 13 pixels.
+Examples of the resulting noisy-labeled segmentations are shown in Figure 1.
+
+![](/article/images/PickAndLearn/figure1.png)
+
+
+# Results
+Table 1 and Figure 3 compare the performance of the baseline method and their proposed architecture when faced with
+more or less noisy labels, and with varying degrees of noise.
+
+![](/article/images/PickAndLearn/table1.png)
+
+![](/article/images/PickAndLearn/figure3.png)
+
+Figure 4 gives some insight into how the QAM affects the final loss during the training.
+
+![](/article/images/PickAndLearn/figure4.png)
+
+
+# Remarks
+- Approximative English and figure make it unclear whether the QAM uses the original segmentation loss or the
```

Comment:
figure : figures

 Approximative English : Unclear wording at some points (?)

Reply:
I reworked my remark to use your suggestion and to specify which single figure in particular I was referring to.

Reply:
Done in 6b69fdd. It's very clear now. Thanks @nathanpainchaud.